<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="EPBYMINW5030" failures="1" tests="1" name="cos test" time="0.047" errors="0" timestamp="01 Apr 2019 07:42:05 GMT">
  <testcase classname="alex.MyFirstTest" name="cosTest" time="0.01">
    <failure type="java.lang.AssertionError" message="expected [0.5403023058681398] but found [0.8414709848078965]">
      <![CDATA[java.lang.AssertionError: expected [0.5403023058681398] but found [0.8414709848078965]
at alex.MyFirstTest.cosTest(MyFirstTest.java:17)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- cosTest -->
</testsuite> <!-- cos test -->
